README.md

```diff
@@ -108,3 +111,16 @@ In GitHub, that will look like the following:
 If everything passes, feel free to merge. You are now using CI.
 
 This is a pretty simple setup for Travis CI. There's a lot of customization that can be done, so check out https://docs.travis-ci.com/ if you're curious about how it could be used. This is a free service, so if you have a public side project, I'd encourage you to set up Travis CI on it. Even if you don't really need to use it, it's great practice.
+
+## Deliverable
+Go to `https://travis-ci.org/<your github username>/TravisCI/builds`. Take a screenshot including the list of builds. There should be 4 tests:
+* The initial commit containing the .travis.yml file (should pass)
+* The commit adding tests to the .travis.yml file (should pass)
+* The commit adding a broken unit test (should fail)
+* The commit fixing that broken unit test (should pass)
+Your image should look like this (with one less commit - yours should only have 4):
```

Comment:
Should we be less stringent on this? If they mess it up, they'll have a different number of commits. Worried they'll think they have to start over.